### ISSUE TITLE
Bot goes offline after some time

### ISSUE THREAD

**Issue by davidthrana** [WizardCM/Discord-Time-Bot#5]
Hello!

Bot seems to go offline after some time and I'm unsure why this is happening. Any ideas?

**Comment by WizardCM** [comment]
Yeah, the bot expects a stable connection and Discord's API isn't always stable. I've been meaning to rewrite the bot with a newer version of the API and with more checks in place, but unfortunately haven't had the time.
